src/coreComponents/constitutive/thermalConductivity/SinglePhaseThermalConductivityBase.hpp

```diff
@@ -125,12 +127,25 @@ class SinglePhaseThermalConductivityBase : public ConstitutiveBase
   virtual void update( arrayView2d< real64 const > const & porosity ) const
   { GEOS_UNUSED_VAR( porosity ); }
 
+  /**
+   * @brief Update the thermal conductivity state
+   * @param[in] temperature the  temperature field
+   */
+  virtual void updateFromTemperature( arrayView1d< real64 const > const & temperature ) const
+  { GEOS_UNUSED_VAR( temperature ); }
+
   /**
    * @brief Getter for the effective conductivities in the subRegion
    * @return an arrayView of effective conductivities
    */
   arrayView3d< real64 const > effectiveConductivity() const { return m_effectiveConductivity; }
 
+  /**
+   * @brief Getter for the derivative of effective conductivities in the subRegion w.r.t. temperature
+   * @return an arrayView of derivative of effective conductivities w.r.t. temperature
+   */
+  arrayView3d< real64 const > dEffectiveConductivity_dT() const { return m_dEffectiveConductivity_dT; }
```

Comment:
do you need this?

Reply:
Yes, absolutly, adding this term is the core of this implementation for temperature dependent single phase thermal conductivity.

Reply:
I meant "do you need this getter"?  We have specific ways to grab fields from consitutive models through `getField`